### PR TITLE
docs: rebranding of Che4z content to remove "CA" from product names

### PR DESCRIPTION
**Pull request by zeibura**
## What does this pull request change?
Rebranding of Che4z content to remove "CA" from product names. Broadcom mainframe products no longer have CA in their names.

## What issues does this pull request fix or reference?
https://github.com/eclipse/che/issues/20983

### CODE REVIEW

**Comment by zeibura** [comment]
Added another commit to attempt to fix the malformatted bullet list under the Zowe CLI section (I think it was because a space was missing before the list)

**Comment by max-cx** [comment]
@zeibura, :smiley: thank you so much for this PR!

**Comment by max-cx** [comment]
For the Che docs team:
open-source > open source
Unix > UNIX
synergy > combination
The rest of the language errors are addressed in https://github.com/Vale-at-Red-Hat/vale-at-red-hat/issues/145.

**Comment by zeibura** [comment]
All addressed

File: modules/extensions/partials/assembly_eclipse-che4z.adoc [hunk 1]
```diff
@@ -131,16 +131,17 @@ https://join.slack.com/t/che4z/shared_invite/enQtNzk0MzA4NDMzOTIwLWIzMjEwMjJlOGM
...
 * link:https://docs.zowe.org/stable/user-guide/cli-db2plugin.html[IBM Db2 plug-in for Zowe CLI]
 * link:https://docs.zowe.org/stable/user-guide/cli-cicsplugin.html[IBM CICS plug-in for Zowe CLI]
 
 == Sample Workflows
 
-=== COBOL Language Support using CA Endevor Bridge for Git and Explorer for Endevor
+=== COBOL Language Support using Endevor Bridge for Git and Explorer for Endevor
 
-* Use VS Code’s built-in Git interface to clone source code from CA Endevor using Endevor’s Bridge for Git.
+* Use VS Code’s built-in Git interface to clone source code from Endevor using Endevor’s Bridge for Git.
```
Comment (themr0c) [inline modules/extensions/partials/assembly_eclipse-che4z.adoc]:
```suggestion
* Use Visual Studio Code’s built-in Git interface to clone source code from Endevor using Endevor’s Bridge for Git.
```

File: modules/extensions/partials/assembly_eclipse-che4z.adoc [hunk 2]
```diff
@@ -131,16 +131,17 @@ https://join.slack.com/t/che4z/shared_invite/enQtNzk0MzA4NDMzOTIwLWIzMjEwMjJlOGM
...
 == Sample Workflows
 
-=== COBOL Language Support using CA Endevor Bridge for Git and Explorer for Endevor
+=== COBOL Language Support using Endevor Bridge for Git and Explorer for Endevor
 
-* Use VS Code’s built-in Git interface to clone source code from CA Endevor using Endevor’s Bridge for Git.
+* Use Visual Studio Code’s built-in Git interface to clone source code from Endevor using Endevor’s Bridge for Git.
 * Edit COBOL code using COBOL Language Support, taking advantage of all the coding assistance provided by the extension.
 * Locate all copybooks in the Git folder structure, and if needed, download other copybooks used in the source code from the mainframe.
 * Use VS Code’s built-in Git interface to push, pull and merge changes.
```
Comment (themr0c) [inline modules/extensions/partials/assembly_eclipse-che4z.adoc]:
```suggestion
* Use Visual Studio Code’s built-in Git interface to push, pull and merge changes.
```

File: modules/extensions/partials/assembly_eclipse-che4z.adoc [hunk 3]
```diff
@@ -8,7 +8,7 @@ pass:[<!-- vale IBM.Terms = NO -->]
```
Comment (themr0c) [inline modules/extensions/partials/assembly_eclipse-che4z.adoc:8]:
```suggestion

pass:[<!-- vale RedHat.operating-system  = NO -->]

pass:[<!-- vale RedHat.Slash = NO -->]

```

**Review by themr0c — APPROVED** [review]

**Pull request merged**